molgenis/molgenis-emx2
# feat: add additional css to theme

Fixes: #3315

how to test:

Add the directory styling via additional css theme and see it working on every directory page
```
/* removing the built-in nav because it conflicts */
nav[aria-label="breadcrumb"]:not(.directory-nav) {
  display: none;
}
ol.breadcrumb {
  margin-top: 0.25rem !important;
  margin-bottom: 0.25rem !important;
}

/* emx2 style override */
#app > div {
  background-color: white !important;
}
nav.navbar {
  background-color: #e9ecef !important;
}
nav.navbar a.nav-link,
nav.navbar button.btn {
  color: #495057 !important;
  background-color: transparent;
}
nav.navbar a.nav-link:hover,
nav.navbar button.btn:hover {
  color: #ec6707 !important;
}
nav.navbar button.btn.btn-outline-light:not(.border-0) {
  border-color: #495057 !important;
}
nav.navbar button.btn.btn-outline-light:not(.border-0):hover {
  border-color: #ec6707 !important;
  background-color: #ec6707 !important;
  color: #dbedff !important;
}

.filterbar > details.dropdown-button > summary {
  color: #08205c !important;
}
.filterbar > details.dropdown-button[open] > summary,
.filterbar > details.dropdown-button:hover > summary,
.filterbar > details.text-white > summary {
  color: white !important;
}
```

todo:
